internal/types/types.go

```diff
@@ -7,27 +7,32 @@ import (
-	PolicyAnnotations              map[string]string  `json:"policyAnnotations,omitempty" yaml:"policyAnnotations,omitempty"`
-	PolicyLabels                   map[string]string  `json:"policyLabels,omitempty" yaml:"policyLabels,omitempty"`
-	ConfigurationPolicyAnnotations map[string]string  `json:"configurationPolicyAnnotations,omitempty" yaml:"configurationPolicyAnnotations,omitempty"`
+	Categories                     []string            `json:"categories,omitempty" yaml:"categories,omitempty"`
+	Controls                       []string            `json:"controls,omitempty" yaml:"controls,omitempty"`
+	CopyPolicyMetadata             bool                `json:"copyPolicyMetadata,omitempty" yaml:"copyPolicyMetadata,omitempty"`
+	Dependencies                   []PolicyDependency  `json:"dependencies,omitempty" yaml:"dependencies,omitempty"`
+	Description                    string              `json:"description,omitempty" yaml:"description,omitempty"`
+	ExtraDependencies              []PolicyDependency  `json:"extraDependencies,omitempty" yaml:"extraDependencies,omitempty"`
+	Placement                      PlacementConfig     `json:"placement,omitempty" yaml:"placement,omitempty"`
+	Standards                      []string            `json:"standards,omitempty" yaml:"standards,omitempty"`
+	ConsolidateManifests           bool                `json:"consolidateManifests,omitempty" yaml:"consolidateManifests,omitempty"`
+	OrderManifests                 bool                `json:"orderManifests" yaml:"orderManifests"`
+	Disabled                       bool                `json:"disabled,omitempty" yaml:"disabled,omitempty"`
+	IgnorePending                  bool                `json:"ignorePending,omitempty" yaml:"ignorePending,omitempty"`
+	InformGatekeeperPolicies       bool                `json:"informGatekeeperPolicies,omitempty" yaml:"informGatekeeperPolicies,omitempty"`
+	InformKyvernoPolicies          bool                `json:"informKyvernoPolicies,omitempty" yaml:"informKyvernoPolicies,omitempty"`
+	GeneratePolicyPlacement        bool                `json:"generatePolicyPlacement,omitempty" yaml:"generatePolicyPlacement,omitempty"`
+	GeneratePlacementWhenInSet     bool                `json:"generatePlacementWhenInSet,omitempty" yaml:"generatePlacementWhenInSet,omitempty"`
+	PolicySets                     []string            `json:"policySets,omitempty" yaml:"policySets,omitempty"`
+	PolicyAnnotations              map[string]string   `json:"policyAnnotations,omitempty" yaml:"policyAnnotations,omitempty"`
+	PolicyLabels                   map[string]string   `json:"policyLabels,omitempty" yaml:"policyLabels,omitempty"`
+	ConfigurationPolicyAnnotations map[string]string   `json:"configurationPolicyAnnotations,omitempty" yaml:"configurationPolicyAnnotations,omitempty"`
+	HubTemplateOptions             *HubTemplateOptions `json:"hubTemplateOptions,omitempty" yaml:"hubTemplateOptions,omitempty"`
```

Comment:
Looks like this pointer is copy/paste, but does it need to be a pointer when it has `omitempty`?

Reply:
It doesn't have to be and seems to work well without it being a pointer. Updated!